core/domain/exp_services_test.py

```diff
@@ -5189,6 +5528,121 @@ def test_get_all_exploration_summaries(self):
         # Check actual summaries equal expected summaries.
         self.assertItemsEqual(actual_summaries, expected_summaries)
 
+    def test_get_top_rated_exploration_summaries(self):
+        exploration_summaries = (
+            exp_services.get_top_rated_exploration_summaries(3))
+        top_rated_summaries = (
+            exp_models.ExpSummaryModel.get_top_rated(3))
+        top_rated_summaries_model = (
+            exp_fetchers.get_exploration_summaries_from_models(
+                top_rated_summaries))
+        self.assertItemsEqual(exploration_summaries, top_rated_summaries_model)
+
+    def test_get_recently_published_exp_summaries(self):
+        self.save_new_valid_exploration(self.EXP_0_ID, self.owner_id)
+        self.save_new_valid_exploration(self.EXP_1_ID, self.owner_id)
+        self.save_new_valid_exploration(self.EXP_2_ID, self.owner_id)
+        rights_manager.publish_exploration(self.owner, self.EXP_0_ID)
+        rights_manager.publish_exploration(self.owner, self.EXP_1_ID)
+        rights_manager.publish_exploration(self.owner, self.EXP_2_ID)
+        exploration_summaries = (
+            exp_services.get_recently_published_exp_summaries(3)
+        )
+        recently_published_summaries = (
+            exp_models.ExpSummaryModel.get_recently_published(3))
+        recently_publshed_summaries_model = (
+            exp_fetchers.get_exploration_summaries_from_models(
+                recently_published_summaries))
+        self.assertEqual(len(exploration_summaries), 3)
+        self.assertItemsEqual(
+        exploration_summaries,
+        recently_publshed_summaries_model)
```

Comment:
I think this should be indented one level. Please check the indentation throughout since I've seen this a number of times

Reply:
i have rechecked the code and corrected the indentation. Thanks for pointing out